src/app/loadout/LoadoutDrawerContents.tsx

```diff
@@ -41,6 +41,7 @@ const loadoutTypes = [
   'Ship',
   'Ships',
   'Vehicle',
+  'Inventory',
```

Comment:
I don't think we need to add this - this is the allow list of item types that can be added to loadouts.